src/components/pagination/page/page.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import injectSheet from 'react-jss';
+import cn from 'classnames';
+import styles from './page-styles';
+
+const Page = ({ classes, isSelected, isFirst, isLast, pageNumber, clickHandler, children }) => (
+  <li
+    className={cn(classes.item, {
+      [classes.itemAlwaysVisible]: isSelected || isFirst || isLast,
+    })}
+  >
+    <button
+      type="button"
+      className={cn(classes.button, {
+        [classes.buttonSelected]: isSelected,
+      })}
+      onClick={() => {
```

Comment:
This function (eg. `() => { clickHandler(pageNumber) }`) should be a class-method instead (requires `Page` to be a class) so that it is not re-created on each render